addons/stock/static/src/less/report_location_barcode.less

```diff
@@ -1,4 +1,7 @@
 .page_stock_location_barcodes {
+    position: relative;
+    width: 210mm;
```

Comment:
That corresponds to A4 paper size, but one could print on Letter paper (US) or any non-A4 paper size.